label_studio/ml_models/api.py

```diff
@@ -223,3 +225,51 @@ def get(self, *args):
                 )
         result = {'projects': compatible_project_list}
         return Response(result, status=200)
+
+
+@method_decorator(
+    name='post',
+    decorator=swagger_auto_schema(
+        tags=['Models: Model Runs'],
+        operation_summary='Create Model Run object',
+        operation_description='Create a new Model Run for given project_id, model_version_id and project_subset if it does not exist, if it exists delete the record and create new',
+    ),
+)
+@method_decorator(
+    name='get',
+    decorator=swagger_auto_schema(
+        tags=['Models: Model Runs'],
+        operation_summary='get list of model runs',
+        operation_description='get list of model runs for the organization',
+    ),
+)
+class ModelRunAPI(generics.ListCreateAPIView):
+    serializer_class = ModelRunSerializer
+    permission_required = ViewClassPermission(  # TODO use same permissions or different?
```

Comment:
It's a good question. I think this is right for now, with the assumption that access to a Model is defined at the org level (plus the role). This seems right to me, but happy to discuss more.

Reply:
Can remove this comment now I think since we are using new permissions @forum-hs